hack/install-kcp-with-plugins.sh

```diff
@@ -19,11 +19,12 @@
 # This script installs KCP binaries and plugins to a folder of choice
 #
 # Arguments:
-# [--version release_version] set a specific KCP release version, default: latest
+# [--version release] set a specific KCP release version, default: latest
```

Comment:
What is the line that says "# Usage: $0 --create-folder --verbose" for?  It omits the vast majority of the options.

Reply:
Again I consider a default/example usage. In most cases users will not specify arch/os and use the default.
Probably I'm miss understanding the purpose of this line